### PR TITLE
Rename Workflow to WorkflowExecution

### PR DESCRIPTION
**Pull request by rushtehrani**

### CODE REVIEW

File: api/workflow.proto [hunk 1]
```diff
@@ -9,52 +9,52 @@ import "metric.proto";
 
 service WorkflowService {
     // Creates a Workflow
-    rpc CreateWorkflow (CreateWorkflowRequest) returns (Workflow) {
+    rpc CreateWorkflowExecution (CreateWorkflowExecutionRequest) returns (WorkflowExecution) {
         option (google.api.http) = {
-            post: "/apis/v1beta1/{namespace}/workflows"
-            body: "workflow"
+            post: "/apis/v1beta1/{namespace}/workflow_exeuctions"
```
Comment (aleksandrmelnikov) [inline api/workflow.proto]: @rushtehrani Typo?

Comment (rushtehrani) [reply api/workflow.proto]: Good catch! there may be more of these.

Comment (aleksandrmelnikov) [reply api/workflow.proto]: That's the only one I've caught so far, maybe Andrey will see one more :)

**Comment by aleksandrmelnikov** [comment]
Approval overall, just that typo.

File: api/workflow.proto (hunk 1 above)
Comment (rushtehrani) [reply api/workflow.proto]: Fixed

**Pull request merged**